pyproject.toml

```diff
@@ -101,6 +102,7 @@ sphinx_rtd_theme = { version = "*", optional = true }
 autodoc_pydantic = { version = "*", optional = true }
 sphinx-reredirects = { version = "^0.1.2", optional = true }
 sphinx-automodapi = { version = "0.16.0", optional = true }
+mistralai = "^0.1.3"
```

Comment:
remove from global dependencies

Reply:
thanks for the proposal. I wanted to do that initially. but when I did, the PR failed some tests in the CI/CD because of all the imports from mistralai client didn't work. Can you recommend a workaround ?

Reply:
I believe the local imports from above should fix this!